test/index.html

```diff
@@ -17,16 +17,12 @@
   <body>
     <script>
       WCT.loadSuites([
-        'app-route-converter.html?dom=shadow',
-        'app-route.html?dom=shadow',
-        'app-location.html?dom=shadow',
-        'test-observer-app.html?dom=shadow',
-        'test-app-example-1.html?dom=shadow',
-        'app-route-converter.html?wc-ce=true&wc-shadydom=true',
-        'app-route.html?wc-ce=true&wc-shadydom=true',
-        'app-location.html?wc-ce=true&wc-shadydom=true',
-        'test-observer-app.html?wc-ce=true&wc-shadydom=true',
-        'test-app-example-1.html?wc-ce=true&wc-shadydom=true'
+        'app-route-converter.html',
+        'app-route.html',
+        'app-location.html',
+        'test-observer-app.html',
+        'test-app-example-1.html',
+        'app-route-optional-params.html'
```

Comment:
Please revert the changes to the test configuration to make sure we do not regress.